ESP32-WiFi-Hash-Monster/ESP32-WiFi-Hash-Monster.ino

```diff
@@ -254,39 +280,47 @@ void setup() {
   // Create a sprite for the header
   header.setColorDepth(1);
   header.createSprite(320, 20);
-  header.setBitmapColor(TFT_WHITE, TFT_BLUE);
-  header.fillSprite(TFT_BLUE);
   header.setFreeFont(FM9);
-  header.setTextDatum(TR_DATUM);
+  header.setTextColor( TFT_BLACK, TFT_BLUE ); // unintuitive: use black/blue mask
+  header.setTextDatum(TL_DATUM);
+  header.setBitmapColor( TFT_BLUE, TFT_WHITE ); // mask will be converted to this color
+  header.fillSprite(TFT_BLUE);
 
   // Create a sprite for the footer
   footer.setColorDepth(1);
   footer.createSprite(320, 40);
-  footer.setBitmapColor(TFT_WHITE, TFT_BLUE);
-  footer.fillSprite(TFT_BLUE);
   footer.setFreeFont(FM9);
-  footer.setTextDatum(TR_DATUM);
+  footer.setTextColor( TFT_BLACK, TFT_BLUE ); // unintuitive: use black/blue mask
+  footer.setTextDatum(TL_DATUM);
+  footer.setBitmapColor( TFT_BLUE, TFT_WHITE ); // mask will be converted to this color
+  footer.fillSprite(TFT_BLUE);
 
   // Create a sprite for the graph1
   graph1.setColorDepth(8);
   graph1.createSprite(128+50, 61); // graph1.pushSprite( 90, 138 );
+  graph1.setTextColor(TFT_WHITE,TFT_BLACK);
+  graph1.setFreeFont(FM9);
   graph1.fillSprite(TFT_BLACK);
 
   // Create a sprite for the graph2
   graph2.setColorDepth(1);
   graph2.createSprite(MAX_X-40, 100);
-  graph2.setBitmapColor(TFT_ORANGE, TFT_BLACK);
+  graph2.setBitmapColor( TFT_GREEN, TFT_BLACK );
```

Comment:
@G4lile0 this achieves the same effect as your last push, however it applies to any device (M5Fire, M5Core2, M5Core classic, Odroid-Go)